lib/icinga/externalcommandprocessor.cpp

```diff
@@ -1101,13 +1120,24 @@ void ExternalCommandProcessor::DelDowntimeByHostName(double, const std::vector<S
 		Log(LogWarning, "ExternalCommandProcessor")
 			<< ("Ignoring additional parameters for host '" + arguments[0] + "' downtime deletion.");
 
+	m_DownTimeRemoved = false;
```

Comment:
I guess this should be reset to `false` for each iteration of the loop, otherwise, if removing of one downtime failed, the log message won't be generated for any following downtime, even if removing that one succeeded.